tee-worker/core/parentchain/indirect-calls-executor/src/lib.rs

```diff
@@ -33,5 +33,11 @@ pub mod sgx_reexport_prelude {
 }
 pub mod error;
 pub mod indirect_calls_executor;
-
+pub mod litentry;
 pub use indirect_calls_executor::*;
+
+pub enum ExecutionStatus {
+	Success,
+	NextExecutor,
+	Skip,
```

Comment:
I don't see the situation when using the `Skip`. It can be removed, if we need it, add it again.